content/project/conduct.adoc

```diff
@@ -7,180 +7,252 @@ section: conduct
 :toc:
 
 
-This document only defines the Code of Conduct and Reporting/Handling processes
-for the Jenkins project/community. For more details about our cultural values,
+This document defines the Code of Conduct and Reporting/Handling processes
+for the Jenkins community. For more details about our cultural values,
 goals, philosophies, structure and so on, please consult our
 link:/project/governance[Governance Document]. 
 
 
-== Code of Conduct
+== Our Pledge
 
-As contributors and maintainers of this project (defined under <<Community Spaces>>
-and the
-link:/project/governance[Governance
-Document]), and in the interest of fostering an open and welcoming community,
-we pledge to respect all people who contribute through reporting issues,
+// TODO(oleg_nenashev): First paragraph is an adapted paragraph of the Contributor Covenant 1.3.0. Should we keep it or remove it?
```

Comment:
We should remove it. It's fairly duplicative and doesn't serve much value with the revised Contributor Covenant. It's better to start with the start of the improved Covenant.

Reply:
I will probably move some bits up so that get displayed before the "Pledge" section. Agreed with removing and keeping pristine Contributor Covenant entry